### ISSUE TITLE
Build Error on release and devel => maybe PubChem & CACTUS issues (win only?)

### ISSUE THREAD

**Issue by schymane** [MassBank/RMassBank#250]
http://bioconductor.org/checkResults/devel/bioc-LATEST/RMassBank/

http://bioconductor.org/checkResults/release/bioc-LATEST/RMassBank/

Although the fail is in the XCMSvignette on the (I thought now deprecated?!) molfiles: 

> Error: processing vignette 'RMassBankXCMS.Rnw' failed with diagnostics:
>  chunk 10 
> Error in createMolfile(as.numeric(c[[1]][["COMMENT"]][[getOption("RMassBank")$annotations$internal_id_fieldname]])) : 
>   Pubchem and Cactus both seem to be down.

... the cause may in fact be more related to an updated TLS protocol... all PubChem functions currently return NAs for me, and this appears also in the build report - NCBI protocol now requires TLS 1.2 or higher

> url <- "https://pubchem.ncbi.nlm.nih.gov/rest/pug/compound/cid/2735102/cids/JSON?cids_type=component"
> timeout <- 60
> getURL(URLencode(url),timeout=timeout)
>Error in function (type, msg, asError = TRUE)  : 
>  error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version

> mbWorkflow: Step 1. Gather info from several databases
> The string does not look like a valid InChIKey: NA
> :1: parser error : Start tag expected, '<' not found
> The string does not look like a valid InChIKey: NA

**Comment by schymane** [comment]
My session info, for the record. We have several users who can confirm these functions work fine on mac/linux but fail on win seemingly irrespective of the R / package version - which supports what is observed in the build reports. All my RChemMass and functions related to getPcId fail similarly (returning NA on win, but functioning perfectly on mac/linux).
Any tips to solve most welcome as this issue has broken a LOT of functions ... @sneumann @meowcat @meier-rene @tsufz 
PubChem confirm that the TLS was upgraded to accepting only 1.2 around Sept 1st, which matches our observations. 

```
R version 3.6.1 (2019-07-05)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 17763)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
[3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] RChemMass_0.1.27 RCurl_1.98-1.2   rjson_0.2.20     enviPat_2.4      rsvg_1.3        
 [6] curl_4.2         RMassBank_2.99.2 Rcpp_1.0.2       rcdk_3.4.7.1     rcdklibs_2.0    
[11] rJava_0.9-11    

loaded via a namespace (and not attached):
 [1] tidyselect_1.0.0      purrr_0.3.3           lattice_0.20-38      
 [4] colorspace_1.4-1      stats4_3.6.1          yaml_2.2.0           
 [7] fingerprint_3.5.7     vsn_3.52.0            XML_3.98-1.20        
[10] rlang_0.4.5           pillar_1.4.2          glue_1.3.1           
[13] MSnbase_2.10.1        mzR_2.18.1            BiocParallel_1.18.1  
[16] affy_1.62.0           BiocGenerics_0.30.0   affyio_1.54.0        
[19] foreach_1.4.7         plyr_1.8.4            mzID_1.22.0          
[22] ProtGenerics_1.16.0   zlibbioc_1.30.0       munsell_0.5.0        
[25] pcaMethods_1.76.0     gtable_0.3.0          codetools_0.2-16     
[28] Biobase_2.44.0        IRanges_2.18.3        doParallel_1.0.15    
[31] parallel_3.6.1        itertools_0.1-3       preprocessCore_1.46.0
[34] scales_1.0.0          BiocManager_1.30.10   limma_3.40.6         
[37] S4Vectors_0.22.1      impute_1.58.0         ggplot2_3.2.1        
[40] png_0.1-7             digest_0.6.21         dplyr_0.8.5          
[43] ncdf4_1.16.1          grid_3.6.1            tools_3.6.1          
[46] bitops_1.0-6          magrittr_1.5          lazyeval_0.2.2       
[49] tibble_2.1.3          crayon_1.3.4          pkgconfig_2.0.3      
[52] MASS_7.3-51.4         assertthat_0.2.1      httr_1.4.1           
[55] rstudioapi_0.10       iterators_1.0.12      R6_2.4.1             
[58] MALDIquant_1.19.3     compiler_3.6.1     
```

**Comment by schymane** [comment]
We had discussed transitioning to webchem, for instance the following works: 

```
> query <- "BSYNRYMUTXBXSQ-UHFFFAOYSA-N"
> library(webchem)
Help us improve the package
Fill out our 2020 survey at https://forms.gle/V7dfGGn73dkesn5L6
> ?webchem
> get_cid(query, from="inchikey")
Querying BSYNRYMUTXBXSQ-UHFFFAOYSA-N. OK (HTTP 200).
# A tibble: 1 x 2
  query                       cid  
  <chr>                       <chr>
1 BSYNRYMUTXBXSQ-UHFFFAOYSA-N 2244 
> as.character(get_cid(query, from="inchikey"))
Querying BSYNRYMUTXBXSQ-UHFFFAOYSA-N. OK (HTTP 200).
[1] "BSYNRYMUTXBXSQ-UHFFFAOYSA-N" "2244"                       
> as.character(get_cid(query, from="inchikey",verbose=F))
[1] "BSYNRYMUTXBXSQ-UHFFFAOYSA-N" "2244"                       
> as.character(get_cid(query, from="inchikey",verbose=F))[1]
[1] "BSYNRYMUTXBXSQ-UHFFFAOYSA-N"
> as.character(get_cid(query, from="inchikey",verbose=F))[2]
[1] "2244"
> as.numeric(get_cid(query, from="inchikey",verbose=F)[2])
[1] 2244
> getPcId(query)
[1] NA
```

So the question is ... do we patch the current issue or speed up the transition - if the latter, what else would need to change?

```
> sessionInfo()
R version 3.6.1 (2019-07-05)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 17763)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
[3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] webchem_1.1.0    purrr_0.3.3      httr_1.4.1       jsonlite_1.6     RChemMass_0.1.27
 [6] RCurl_1.98-1.2   rjson_0.2.20     enviPat_2.4      rsvg_1.3         curl_4.2        
[11] RMassBank_2.99.2 Rcpp_1.0.2       rcdk_3.4.7.1     rcdklibs_2.0     rJava_0.9-11    

loaded via a namespace (and not attached):
 [1] Biobase_2.44.0        vsn_3.52.0            foreach_1.4.7        
 [4] assertthat_0.2.1      BiocManager_1.30.10   affy_1.62.0          
 [7] stats4_3.6.1          yaml_2.2.0            impute_1.58.0        
[10] pillar_1.4.2          lattice_0.20-38       glue_1.3.1           
[13] limma_3.40.6          fingerprint_3.5.7     digest_0.6.21        
[16] rvest_0.3.5           colorspace_1.4-1      preprocessCore_1.46.0
[19] plyr_1.8.4            MALDIquant_1.19.3     XML_3.98-1.20        
[22] pkgconfig_2.0.3       zlibbioc_1.30.0       scales_1.0.0         
[25] itertools_0.1-3       affyio_1.54.0         BiocParallel_1.18.1  
[28] tibble_2.1.3          IRanges_2.18.3        ggplot2_3.2.1        
[31] BiocGenerics_0.30.0   lazyeval_0.2.2        cli_1.1.0            
[34] magrittr_1.5          crayon_1.3.4          data.tree_1.0.0      
[37] ncdf4_1.16.1          fansi_0.4.0           doParallel_1.0.15    
[40] MASS_7.3-51.4         xml2_1.2.2            mzR_2.18.1           
[43] tools_3.6.1           stringr_1.4.0         MSnbase_2.10.1       
[46] S4Vectors_0.22.1      munsell_0.5.0         pcaMethods_1.76.0    
[49] compiler_3.6.1        mzID_1.22.0           rlang_0.4.5          
[52] grid_3.6.1            iterators_1.0.12      rstudioapi_0.10      
[55] bitops_1.0-6          base64enc_0.1-3       gtable_0.3.0         
[58] codetools_0.2-16      R6_2.4.1              dplyr_0.8.5          
[61] utf8_1.1.4            ProtGenerics_1.16.0   stringi_1.4.3        
[64] parallel_3.6.1        vctrs_0.2.4           png_0.1-7            
[67] tidyselect_1.0.0     
```

**Comment by sneumann** [comment]
Fixed with 00101559ae3ab1c841c8026366216abde278d07b. Yours, Steffen
